### PR TITLE
Usability Updates to Make Easier for Customers

### PR DESCRIPTION
**Pull request by seanjw13**
- Parameterized load tests (PATs, host, endpoint name)
- Added requirements section
- Added intro section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- [Record the instance name](https://docs.databricks.com/workspace/workspace-details.html#workspace-instance-names-urls-and-ids) of the Databricks workspace where the model is deployed.
+- Databricks APIs use Personal Access Tokens (PAT)s to authenticate. Use an existing PAT or [generate a new PAT](https://docs.databricks.com/dev-tools/auth.html#personal-access-tokens-for-users) that has "Can View" or "Can Manage" permissions on the model serving endpoint. 
+- If your Databricks workspace restricts IP addresses to an IP access list, your load testing source's IP address must be within the access list. You may need to connect to your company's VPN. You can check if IP access lists are used in your Databricks workspace by [following the instructions here](https://docs.databricks.com/security/network/ip-access-list.html#check-if-your-workspace-has-the-ip-access-list-feature-enabled).
+
+## Setting up Locust on your own computer in single-process mode (5 min setup, supports low concurrency)
```
Comment (lichenran1234) [inline README.md]: `low QPS` rather than `low concurrency`

File: locust.md [hunk 2]
```diff
@@ -1,134 +0,0 @@
-# Quickly setting up a load test client using Locust
```
Comment (lichenran1234) [inline locust.md:1]: Can we not delete this file? I already shared the `https://github.com/lichenran1234/load-test/blob/main/locust.md` link to some customers (my fault), and that link will be invalidated if this file is removed.

Instead, let's keep this file but modify the content to be `Please refer to README.md`?

**Review by lichenran1234 — COMMENTED** [review]
Looks much better than my original version!! Thanks! Just some nits, otherwise looks good

File: locust.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
+__Note:__ There are more up to date instructions in the [README.md](README.md) file. Please use that as a reference going forward.
+
 # Quickly setting up a load test client using Locust
```
Comment (lichenran1234) [inline locust.md]: Thanks a lot! I think we can delete everything except the `Note` above, what do you think?

Comment (seanjw13) [reply locust.md]: No problem! Yeah, let's go ahead and delete the rest since it's redundant.

File: locust.md (hunk 2 above)
Comment (seanjw13) [reply locust.md:1]: Makes sense. I added it back in.

File: README.md (hunk 1 above)
Comment (seanjw13) [reply README.md]: Made the change.

**Comment by seanjw13** [comment]
I made the recommended change to locust.md. Let me know if you see any other changes that need to be made.

**Review by lichenran1234 — APPROVED** [review]
Thanks so much!!!

**Pull request merged**